lib/tapioca/compilers/dsl/active_record_columns.rb

```diff
@@ -380,12 +379,7 @@ def lookup_return_type_of_method(column_type, method)
         sig { params(column_type: ActiveModel::Type::Value, method: Symbol).returns(T.nilable(String)) }
         def lookup_arg_type_of_method(column_type, method)
           signature = T::Private::Methods.signature_for_method(column_type.method(method))
-          return unless signature
-
-          arg_type = signature.arg_types.first.last
-          return if T::Types::Simple === arg_type && T::Generic === arg_type.raw_type
-
-          arg_type.to_s
+          signature.arg_types.first.last.to_s if signature
```

Comment:
Do you mind using intermediate variable with meaningful names so we can understand what we're getting from the `first` then `last`?